### ISSUE TITLE
csi-snapshotter broke in rook v1.16.0 with upgraded volumesnapshot crds

### ISSUE THREAD

**Issue by buroa** [rook/rook#15195]
**Is this a bug report or feature request?**
* Bug Report

**Deviation from expected behavior:**
csi-snapshotter containers are crashing with:

```
flag provided but not defined: -feature-gate
Usage of /csi-snapshotter:
```

**Expected behavior:**
container not crashing

**How to reproduce it (minimal and precise):**
Upgrade to rook-ceph v1.16.0 with upgraded `volumegroupsnapshotclasses` at `v1beta1`

**Logs to submit**:
```
flag provided but not defined: -feature-gate
Usage of /csi-snapshotter:
```

**Comment by travisn** [comment]
The fix has been backported to the 1.16 branch. For clusters where this is a blocking issue, until the v1.16.1 release is out, you can use this rook operator image: `rook/ceph:v1.16.0-4.g6da8ef89d`.

**Comment by jfroy** [comment]
> The fix has been backported to the 1.16 branch. For clusters where this is a blocking issue, until the v1.16.1 release is out, you can use this rook operator image: `rook/ceph:v1.16.0-4.g6da8ef89d`.

You probably also need to set `csi.snapshotter.tag=v8.2.0` in the operator chart values.
